designer/client/page-create.js

```diff
@@ -8,8 +8,9 @@ class PageCreate extends React.Component {
     e.preventDefault()
     const form = e.target
     const formData = new window.FormData(form)
-    const path = formData.get('path').trim()
     const { data } = this.props
+    const title = formData.get('title').trim()
+    const path = '/'.concat(title.replace(/[^a-zA-Z ]/g, '').replace(' ', '-')).toLowerCase()
```

Comment:
This is fine, so just a note worth knowing - this will strip non-English characters, e.g. á.